api/catalog/api/serializers/media_serializers.py

```diff
@@ -243,6 +252,19 @@ def validate_unstable__sort_dir(self, value):
     def validate_unstable__authority(self, value):
         return False if self.is_request_anonymous() else value
 
+    def validate_unstable__include_sensitive_results(
+        self,
+        value,
+    ):
+        exclusive_fields = ("mature", "unstable__include_sensitive_results")
+        if all(f in self.initial_data for f in exclusive_fields):
+            raise serializers.ValidationError(
+                "`mature` and `unstable__include_sensitive_results` "
+                "must not both be defined."
+            )
+
+        return self.initial_data.get("mature") or value
```

Comment:
What is doing `mature` at this point if they are exclusive?

Reply:
`mature` will work as a deprecated alias for `include_sensitive_results` so that API consumers currently relying on the `mature` query will not have their results break but will still get more expansive information about sensitive content generally. This was discussed in the implementation plan and in the project proposal:

https://docs.openverse.org/projects/proposals/trust_and_safety/detecting_sensitive_textual_content/20230330-implementation_plan_filtering_and_designating_results_with_sensitive_textual_content.html#query-the-filtered-index-overview-step-3

and 

https://docs.openverse.org/projects/proposals/trust_and_safety/detecting_sensitive_textual_content/20230309-project_proposal_detecting_sensitive_textual_content.html#filtering

> Results with sensitive terms in their textual content are only included in results when include_sensitive_results=True. This parameter will supplant the mature=True as a more comprehensively and descriptively named parameter. mature=True should still work, but it should just be an alias for include_sensitive_results and should be marked as deprecated in or altogether excluded from the API documentation.